tests/aeroval/test_experiment_output.py

```diff
@@ -113,7 +113,7 @@ def test_ExperimentOutput():
 def test_ExperimentOutput_error():
     with pytest.raises(ValueError) as e:
         ExperimentOutput(None)
-    assert str(e.value) == "need instance of <class 'pyaerocom.aeroval.setupclasses.EvalSetup'>"
+    assert str(e.value) == "need instance of <class 'pyaerocom.aeroval.setup_classes.EvalSetup'>"
```

Comment:
I think this `ValueError.value` comes from https://github.com/metno/pyaerocom/blob/be7aeeff06f5ea49b38c3a717567c4a516beacf2/pyaerocom/_lowlevel_helpers.py#L105

so it would be a little tricky to change just in this case.

Reply:
I just meant replace 
```python
assert str(e.value) == "need instance of <class 'pyaerocom.aeroval.setupclasses.EvalSetup'>"
```
with
```python
assert str(e.value) == f"need instance of {EvalSetup}"
```
This would not need any changes to _lowlevel_helpers.py. The string expansion of `{self._type}` and `{class}` is the the same.